apps/site/i18n/locales/en.json

```
@@ -290,6 +290,7 @@
         "platform": "Platform"
       },
       "codeBox": {
+        "fnmEnvSetup": "Configure fnm environment with `fnm env --use-on-cd`",
```

```suggestion
        "fnmEnvSetup": "configure fnm environment",
```
Perhaps we can shorten the comment to this? The others are all also lowercase so I've done that for consistency here. 

This is a nit though - I'm happy for this to land as is